recipes/parso/bld.bat

```diff
@@ -0,0 +1,2 @@
+"%PYTHON%" setup.py install --single-version-externally-managed --record=record.txt
```

Comment:
If you have the `script: setup.py install --single-version-externally-managed --record=record.txt` in the `meta.yaml`, you can delete the `build.sh` and `bld.bat` files.